website/client/src/components/groups/groupFormModal.vue

```diff
@@ -540,7 +540,6 @@ export default {
       if (!this.workingGroup.name) errors.push(this.$t('nameRequired'));
       if (!this.workingGroup.summary) errors.push(this.$t('summaryRequired'));
       if (this.workingGroup.summary.length > MAX_SUMMARY_SIZE_FOR_GUILDS) errors.push(this.$t('summaryTooLong'));
-      if (!this.workingGroup.description) errors.push(this.$t('descriptionRequired'));
```

Comment:
thanks @akmaekki can you revert this line? I'm saying this because while the error is not used anymore when updating parties it's still used when creating one, see line https://github.com/HabitRPG/habitica/pull/11499/files#diff-5b256091a52b128af86f73ea9ec9e4e0R250

Reply:
@paglias 
Easy to revert, but for me it seems illogical that a description has to be specified when a party is created but can be empty when party name changes.

Reply:
agreed but that modal is also used by guilds where description is mandatory

Reply:
Ah, I see. Thanks for the explanation.